feature/system/tests/system_base_test/g_protocol_test.go

```diff
@@ -58,7 +59,14 @@ func TestGNMIClient(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	conn := dialConn(t, dut, introspect.GNMI, 9339)
 	c := gpb.NewGNMIClient(conn)
-	if _, err := c.Get(context.Background(), &gpb.GetRequest{Encoding: gpb.Encoding_JSON_IETF, Path: []*gpb.Path{{Elem: []*gpb.PathElem{}}}}); err != nil {
+	if _, err := c.Get(context.Background(), &gpb.GetRequest{
```

Comment:
get on root [an empty pathelem] is a valid operation according to the gnmi spec. 

if an implementation does not support it, IMO it should be a deviation.

cc @robshakir @dplore

Reply:
Agreed on this item.  @kjahed your suggestion isn't invalid, but I would like to see a request to the empty path succeed.

Reply:
@dplore @LimeHat - can you elaborate on what is the need for probing empty path; 

@kjahed - can you respond on this query as to whether gnmi get for empty path is supported on not?

Reply:
Regarding the need to do a query, see step 3:

`3. Validate client properly connects and execute a simple RPC to validate no errors are returned.`

Root is a good choice of query for this as it's schema independent and required by the gnmi specification.

Reply:
@ram-mac we do not support Get on root/empty path. Will add a deviation.

@dplore totally agree that that's what the spec requires. The reason a deviation was not added in the first place is that the purpose of the test (per the readme) is to test that g* services work, not that gNMI get on a specific path works.

Reply:
That's a fair comment.  The README is vague about the precise query that should be used.     In this case, I would like to see a deviation.